### PR TITLE
Extend Plotly support for all chart types

### LINKED ISSUE #782: Extended support for plotly diagrams + Improved scope documentation of kedro's plotly support
Hey team, 

thanks for the great tool.

Cross posting https://github.com/kedro-org/kedro/issues/1337.

## Description

I am trying to:
1. save a sankey diagram in Kedro
1. visualise a sankey diagram in Kedro viz. 
[note: turns out that this is not possible]

## Context

- The best way to visualize the output of the pipeline is a Sankey diagram.
- Unfortunately, neither`Plotly.JSONDataset` nor `plotly.PlotlyDataSet` support this kind of diagram

**tl;dr**: Currently Kedro Viz is limited to the basic diagrams that plotly.js includes in its basic version. 

## Possible Implementation

Overall, **my request** is:
- extend kedro's support non basic `plotly.*Dataset` (e.g. `HTMLdataset`)
- extend kedro-viz support accordingly
Seems to me that supporting (plotly) html dataset would make kedro/kedro viz much more versatile for diagram vi

## Possible Alternatives
**In the meantimes**, that would be nice to make clear in Kedro's doc that the plotly support is still limited and clearly state what can and cannot be done.

## Checklist

- [x] Include labels so that we can categorise your feature request

### PR DESCRIPTION
**Pull request by rashidakanchwala**
## Description

Resolves #782 

## Development notes

Just updated package.json file to include full plotly js library (min version) 

## QA notes

Tested this with a kedro project where you create a Sankey chart and now the Sankey chart shows up. 

## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/853"></a>

### CODE REVIEW

**Review by studioswong — APPROVED** [review]
great stuff!

perhaps worth updating our readme to mention that Kedro-viz now supports the visualization of all plots as outlined [here](https://plotly.com/javascript/)?

**Review by tynandebold — APPROVED** [review]
Looks great, just a couple small things from me :)

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "lodash.debounce": "^4.0.8",
     "lodash.noop": "^3.0.1",
     "lodash.sortby": "^4.7.0",
-    "plotly.js-cartesian-dist": "^1.58.4",
+    "plotly.js-dist-min": "^1.58.4",
```
[the remark this answers is not in the document]
Comment (rashidakanchwala) [reply package.json:68]: it was breaking actually. hence i kept the old version

[the remark this answers is not in the document]
Comment (tynandebold) [reply package.json:68]: Roger that. Thanks for checking.

**Pull request merged**